Zend/tests/traits/bug74872.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+Bug #74872 (First Trait wins on importing same methods with diff alias)
+--FILE--
+<?php
+
+trait Trait1
+{
+    public function init()
+    {
+        echo("Trait1 - init\n");
+    }
+}
+
+trait Trait2
+{
+    public function init()
+    {
+        echo("Trait2 - init\n");
+    }
+}
+
+class Test
+{
+    use Trait1 {
```

Comment:
What happens if we make this `use Trait1, Trait3` and `Trait3` also has an init method? If I'm reading your code right, it's just going to pick one of them. But I think this should be an error.

I'm wondering if an absolute trait method reference shouldn't be required when multiple traits are used in one block.

Reply:
Right now it still picks one of them in that case. The currrent behavior is to disambiguate when possible and keep the old behavior (pick one) when it's not possible to know. But yes, I think it's a good idea to error in those cases.

I've started porting this to 7.4 but still have to redo the opcache part. So I'll also rework that.